connector/protobuf/src/test/resources/protobuf/functions_suite.proto

```diff
@@ -321,10 +325,10 @@ message Proto3AllTypes {
   NestedEnum enum_val = 3;
   OtherExample message = 4;
 
-  optional int64 optional_int = 5;
-  optional string optional_text = 6;
-  optional NestedEnum optional_enum_val = 7;
-  optional OtherExample optional_message = 8;
```

Comment:
These were just removed because I am using the proto 3 compiler, which prohibits optional fields.